tardis/energy_input/gamma_packet_loop.py

```diff
@@ -322,7 +318,6 @@ def gamma_packet_loop(
         energy_plot_df_rows,
         energy_out,
         deposition_estimator,
-        packets_out,
         bin_width,
```

Comment:
Do we need the `bin_width` as an output? WE can do that later in a post-processing step.

Reply:
Probably not, but the blame said you added it. I can remove it, but it may be out of scope for this PR.